dcat/index.html

```diff
@@ -264,24 +264,20 @@ <h3>DCAT scope</h3>
         A <b>dataset</b> in DCAT is defined as a "collection of data, published or curated by a single agent, and available for access or download in one or more serializations or formats".
         A dataset is a conceptual entity, and can be represented by one or more <b>distributions</b> that serialize the dataset for transfer.
         Distributions of a dataset can be provided via <b>data services</b>.
-        Detailed properties for a data service API are out of the scope of this version of DCAT.
     </p>
 
     <p>
-      A <b>data service</b> in DCAT is a collection of operations or API which provides access to data.
-      An interactive interface may provide access to the API operations.
+      A <b>data service</b> in DCAT is a collection of operations, made available through an API, which provides access to data.
       A data service typically provides selection, extraction, combination, processing or transformation operations over datasets that might be hosted locally or remote to the service.
       A data service might be tied to specific datasets, or its source data might be configured at request- or run-time.
       The result of a request to a data service is a representation of a part or all of a dataset or catalog.
       A data service might represent a data distribution service to select and download a distribution of a dataset or subset, or a data discovery service to find the description of a suitable dataset.
-      Other kinds of data service might include data transformation services such as coordinate transformation services, re-sampling and interpolation services, and various data processing services, but these are outside the scope of the current DCAT vocabulary.
```

Comment:
Don't understand why you want to remove this. Is it misleading or unhelpful?

Reply:
I don't understand why you would put it in and then say that it is out of scope.